src/Widgets/AddressWidget/AddressWidgetComponent.tsx

```diff
@@ -109,22 +106,28 @@ function Table({
       <tbody>
         {lines.map(([name, value]) => (
           <tr key={name}>
-            <td>{`${LOCALIZATION[name]}: `}</td>
+            <td>
+              <ContentTag
+                tag="span"
+                content={widget}
+                attribute={`${name}Label`}
+              />
+              :{' '}
+            </td>
             <td className="text-break">
-              <a href={`${LINK_PREFIXES[name]}:${value}`}>{value}</a>
+              <ContentTag
```

Comment:
Nice!

I only see, that in edit-mode this looks a bit broken design wise:

![Screenshot 2024-04-11 at 14 26 12](https://github.com/Scrivito/scrivito-portal-app/assets/86275/0570870d-b349-41d8-9873-3a9c60990ba8)

Reply:
https://github.com/infopark/scrivito_js/issues/10820